.github/workflows/ci.yml

```diff
@@ -53,3 +53,38 @@ jobs:
 
       - name: Run tests
         run: pnpm coverage
+
+      # - name: Setup TeX Live
+      #   uses: teatimeguest/setup-texlive-action@v3
+      #   with:
+      #     packages: scheme-basic
+
+      - name: Install texlive-latex
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y texlive-latex-base texlive-xetex texlive-latex-extra texlive-fonts-recommended texlive-fonts-extra
```

Comment:
1. let us just use `apt` instead of `apt-get` across the whole github CI flow
2. the minimum requirements for texlive dependency: `sudo apt install -y texlive-xetex texlive-fonts-extra texlive-lang-all`